src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,279 @@
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import { CaretSvg } from '@axa-ch/materials/icons';
+import '@axa-ch/container';
+import { LitElement, html, svg, css, unsafeCSS } from 'lit-element';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+class AXAFooter extends LitElement {
+  static get tagName() {
+    return 'axa-footer';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      dynamic: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.dynamic = false;
+    this.onItemClick = () => {}; // Only for react
```

Comment:
`// Only for react` -> not true, all those languages having functions as callbacks.

Reply:
But none other can use it, since skatejs is enabling this. Skatejs has this onItemClick as props that are handed over. Litelement does currently not keep it as a prop. Only the default value is set here.

Reply:
Right, thx